data/tools/wmllint

```diff
@@ -940,15 +940,13 @@ def pangoize(message, filename, line):
         amper = message.find('&')
         if message[amper:amper+1].isspace():
             message = message[:amper] + "&amp;" + message[amper+1:]
-    rgb = re.search("(?:<|&lt;)([0-9]+),([0-9]+),([0-9]+)(>|&gt;)", message)
+    rgb = re.search("(?:<|&lt;)([0-9]+),([0-9]+),([0-9]+)(?:>|&gt;)", message)
     if rgb:
-        r = int(rgb.group(1))
-        g = int(rgb.group(2))
-        b = int(rgb.group(3))
-        if ( r or g or b) > 255:
+        r, g, b = (int(c) for c in rgb.groups())
+        if any(val > 255 for val in (r, g, b)):
```

Comment:
Good point; I was caught up with fixing the initial bug and didn't think to use `max()`.